Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -183,10 +183,14 @@ bool EventHandler::handle_mousedown(const Gfx::IntPoint& position, unsigned butt
     }
 
     NonnullRefPtr document = *m_frame.active_document();
+    auto result = layout_root()->hit_test(position, Layout::HitTestType::Exact);
+    auto pointer_events = result.layout_node->computed_values().pointer_events();
+    if (pointer_events == CSS::PointerEvents::None)
+        return false;
```

Comment:
This doesn't quite do what you want. Imagine two elements, A and B, with A in front of B, and A has `pointer-events: none`. What we get here is that nothing will happen and we return false, when I believe we want to direct the click event to B. (Though I might be wrong on that, I can't find a good description on how it should work.)

I think checking for `pointer-events` wants to happen inside `hit_test()`, so that nodes can be skipped there. BUT, `hit_test()` is already quite buggy when anything has a `z-index` and there are probably other issues with it. So just adding a FIXME here should be enough for now. (Unless you want to fix hit-testing completely, which would be great!)

Reply:
I've added the comment. I implemented it this way as I didn't want users interacting with iframes with an element on-top with `pointer-events: none`. Currently web browser allow this but not sure if it's a good idea

Reply:
Thanks for the help! :D